hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -996,77 +996,289 @@
     }
   ],
 
-  // ===== INTERRUPT CTRL =====================================================
-
-  // RV_PLIC has two searching algorithm internally to pick the most highest priority interrupt
-  // source. "sequential" is smaller but slower, "matrix" is larger but faster.
-  // Choose depends on the criteria. Currently it is set to "matrix" to meet FPGA timing @ 50MHz
-
-  // ===== ALERT HANDLER ======================================================
+  // ===== PINMUX & PINOUT ======================================================
+
+  pinout: {
+    // IO power bank declaration.
+    // This list defines the IO bank power domains on the ASIC.
+    // Each individual pad must be in one of the declared power domains.
+    banks: ['VCC', 'AVCC', 'VIOA', 'VIOB'],
+    // Pad declaration.
+    // Each entry must have the following four keys:
+    //
+    // - name: Name of the pad (this will be exposed at the chiplevel).
+    //
+    // - type: Pad type (this maps to the pad types defined in prim_pad_wrapper_pkg.sv)
+    //
+    // - bank: Specifies in which of the IO power banks this pad lives.
+    //
+    // - muxed: If this is set to True, the pad will be connected to the pinmux matrix.
+    //          If this is set to False, the pad can be referenced by direct connections in
+    //          the pinmux signal mapping further below.
+    //          If an unmuxed pad is NOT referenced in the pinmux signal mapping,
+    //          the pad needs to be wired up manually in the template.
+    //
+    // Optionally, each pad can also have a 'desc' field for further description.
+    pads: [
+      // TODO: this needs to be updated to the latest ASIC pinout.
+      // Dedicated
+      { name: 'POR_N'           , type: 'InputStd', bank: 'VCC' , muxed: 'False', desc: 'System reset'},
+      { name: 'SPI_HOST_D0'     , type: 'InputStd', bank: 'VIOA', muxed: 'False', desc: 'SPI host data'},
```

Comment:
i think the `SPI_HOST_D*` pins probably should be `BidiStd`

Reply:
ah okay i'm guessing all the pad type changes is something that will be done as a follow-on PR.

Reply:
Ah yeah good point. Looks like a copy-paste error.

Reply:
yeah no worries we can do it as a follow-on. It doesn't impact the prim_generic_pad anyways for now :) 
otherwise i'm sure CI would fail miserably.